File: clients/java_graal/src/main/java/komus24/MyStrategy.java
package komus24;

public class MyStrategy {
    public MyStrategy(komus24.model.Constants constants) {}
    public komus24.model.Order getOrder(komus24.model.PlayerView playerView, DebugInterface debugInterface) {
        return new komus24.model.Order(new komus24.model.VehicleOrder[0]);
    }
    public void debugUpdate(int displayedTick, DebugInterface debugInterface) {}
    public void finish() {}
}

File: clients/swift/Sources/Komus24/MyStrategy.swift
class MyStrategy {
    init(_ constants: Constants) {}
    func getOrder(_ playerView: PlayerView, _ debugInterface: DebugInterface?) -> Order {
        return Order(vehicles: [])
    }
    func debugUpdate(_ displayedTick: Int32, _ debugInterface: DebugInterface) {}
    func finish() {}
}